sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/CertificateClientTestBase.java

```diff
@@ -241,10 +231,18 @@ void getCertificateOperationRunner(Consumer<String> testRunner) {
     @Test
     public abstract void cancelCertificateOperation(HttpClient httpClient, CertificateServiceVersion serviceVersion);
 
+
     void cancelCertificateOperationRunner(Consumer<String> testRunner) {
         testRunner.accept(generateResourceId("testCertificate11"));
     }
 
+    @Test
+    public abstract void cancelCertificateCreationOperation(HttpClient httpClient, CertificateServiceVersion serviceVersion);
+
+    void cancelCertificateCreationOperationRunner(Consumer<String> testRunner) {
+        testRunner.accept(generateResourceId("testCertificate11"));
+    }
```

Comment:
Please refer to my comments on `CertificateClient` on the naming this method and `listPropertyOfCertificates`.